easybuild/easyconfigs/x/x264/x264-20160304-goolf-1.7.20.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'x264'
+version = '20160304'
+
+homepage = 'http://www.videolan.org/developers/x264.html'
+description = """x264 is a free software library and application for encoding video streams into the H.264/MPEG-4 AVC compression format, and is released under the terms of the GNU GPL."""
```

Comment:
@eediaz1987 please break this up into multiple lines, max. line length is 120 columns